phones/models.py

```diff
@@ -76,6 +79,16 @@ def get_valid_realphone_verification_record(user, number, verification_code):
     ).first()
 
 
+def get_last_text_sender(relay_number):
+    return (
+        InboundContact.objects.filter(
+            relay_number=relay_number, last_inbound_type="text"
+        )
+        .order_by("-last_inbound_date")
+        .first()
```

Comment:
Maybe use `latest`?

```suggestion
        .latest("last_inbound_date")
```

Reply:
Aha! That's why we should have non-null fields!! :trollface: